apps/epic-web/src/pages/products/index.tsx

```diff
@@ -129,20 +118,12 @@ const ProductCard: React.FC<{
   product: SanityProduct
   purchase: PurchaseWithProduct | undefined
 }> = ({product, purchase}) => {
-  const router = useRouter()
   const {data: formattedPrice, status: formattedPriceStatus} =
     trpc.pricing.formatted.useQuery({
       productId: product.productId as string,
       quantity: 1,
     })
 
-  const {data: availableBonuses = []} =
-    trpc.bonuses.availableBonusesForPurchase.useQuery({
-      purchaseId: purchase?.id,
-    })
-
-  const {mutate: redeemBonus} = trpc.bonuses.redeemBonus.useMutation()
-
```

Comment:
nice cleanup -- I was about to rip these out after searching for where `redeemBonus` was being used throughout this app